### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BalintMendli**

### CODE REVIEW

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
First part of project was excellent. Since this is project about positioning and floating please add sidebar with editors picks that should have absolute position. 
Please submit new review request.
Happy coding.

**Comment by AlejoCode** [comment]
Thank you so much for the feedback, sidebar was added and a new review request was done as well

File: index.html [hunk 1]
```diff
@@ -13,6 +13,1026 @@
...
+          <span id="fade4">
+            special energy that exerted a repulsive force, causing the universe
+            to swell faster
+          </span>
+          <span id="fade5">
+            than the speed of light for a prodigiously violent instant.</span
+          >
+        </p>
+      </div>
+      <div class="editPicks">
```
Comment (zlayabekrija) [inline index.html:220]: use some absolute positioning to div down, currently it's overlapping the image.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great work so far minor change to be done and you are ready.
Please fix it, and resubmit the request.
Happy coding

File: index.html (hunk 1 above)
Comment (BalintMendli) [reply index.html:220]: Positioning fixed and code review request submitted.

**Review by zlayabekrija — APPROVED** [review]
Excellent work.
Thumbs up 👍 
Happy coding :)

**Pull request merged**